README.md

```diff
@@ -62,6 +62,10 @@ will use it to auto-generate user-visible message in the frontend. , e.g., "You
 | `HOMEPAGE_URL` | `AUTHSERVICE_URL_PREFIX/site/homepage` | Homepage of the application that can be accessed by anonymous users. |
 | `AFTER_LOGOUT_URL` | `AUTHSERVICE_URL_PREFIX/site/homepage` | URL to redirect the user to after they logout. This option used to be called `STATIC_DESTINATION_URL`. For backwards compatibility, the old environment variable is also checked.|
 | `AUTH_HEADER` | `Authorization` | When the AuthService logs in a user, it creates a session for them and saves it in its database. The session secret value is saved in a cookie in the user's browser. However, for programmatic access to endpoints, it is better to use headers to authenticate. The AuthService also accepts credentials in a header configured by the `AUTH_HEADER` setting. |
+| `MULTIPLE_HOSTS` | `false` | Enable auth for multiple hosts. By default, the auth service is hosted on a single subdomain. Setting this to true will also save the original request's host + scheme |
```

Comment:
I would recommend removing this setting and just always saving the host + scheme in the state. Or maybe, as you suggested, doing based on `SESSION_DOMAIN` being set, but I don't really see a reason to do this conditionally.

Reply:
Changed to be based on SESSION_DOMAIN.